contracts/migrations/Migratable.sol

```diff
@@ -29,17 +29,21 @@ contract Migratable {
    */
   mapping (string => mapping (string => bool)) internal migrated;
 
+  string constant private INITIALIZED_ID = "initialized";
+
 
   /**
    * @dev Modifier to use in the initialization function of a contract.
    * @param contractName Name of the contract.
    * @param migrationId Identifier of the migration.
    */
   modifier isInitializer(string contractName, string migrationId) {
+    require(!isMigrated(contractName, INITIALIZED_ID));
     require(!isMigrated(contractName, migrationId));
     _;
     emit Migrated(contractName, migrationId);
     migrated[contractName][migrationId] = true;
+    migrated[contractName][INITIALIZED_ID] = true;
```

Comment:
should we provide a way to store this information for on-chain contracts that have already been initialized? I can be done using a new empty initializer tho

Reply:
Oh, thank you. That was an important part of the fix that I forgot...